admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala

```diff
@@ -174,7 +174,7 @@ object DelegateApiHandler {
     private[this] val mapper = new ObjectMapper with ScalaObjectMapper
     mapper.registerModule(DefaultScalaModule)
     mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
-    mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL)
```

Comment:
Is this related to the finagle change? Do we need to change our jackson version to be consistent with finagle?